tests/native_tests/language.hy

```diff
@@ -274,6 +274,32 @@
   (assert (= (cond) nil)))
 
 
+(defn test-if []
```

Comment:
I see integers, `nil`, and the empty string in this test, but I think we should exercise the paradigmatic conditions `true` and `false` here, too. (For exhaustiveness, we could also test nonempty strings, empty and nonempty collections, floats, and custom classes. This is perhaps pedantic&mdash;I don't _actually_ expect these extra cases to uncover any bugs in the implementation&mdash;but testing paranoia is probably healthy.)